airbyte-connector-builder-server/unit_tests/connector_builder/impl/test_default_api.py

```diff
@@ -155,6 +155,7 @@ def test_read_stream():
     request = {
         "url": "https://demonslayers.com/api/v1/hashiras?era=taisho",
         "headers": {"Content-Type": "application/json"},
+        "http_method": "GET",
```

Comment:
fix tests

Reply:
we'll be able to merge this PR when this is merged in so CI fails on broken tests https://github.com/airbytehq/airbyte/pull/20217

Reply:
Thanks for pointing that out - was looking into it for a PR I'm working on right now 😅